src/overlays/actors/ovl_En_Hata/z_en_hata.c

```diff
@@ -27,15 +26,106 @@ const ActorInit En_Hata_InitVars = {
     (ActorFunc)EnHata_Update,
     (ActorFunc)EnHata_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Hata/EnHata_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Hata/EnHata_Destroy.s")
+static UNK_TYPE4 EnHata_UnusedData[] = { 0x0A000939, 0x20010000, 0x00000000, 0x00000000, 0x00000000,
+                                         0x00000080, 0x00000000, 0x00050100, 0x001000F6, 0x00000000,
+                                         0x00000000, 0x00000000, 0x00000000, 0xFF000000 };
```

Comment:
if you put a trailing comma on the last entry clang wont ruin the formatting
```suggestion
static UNK_TYPE4 EnHata_UnusedData[] = { 
    0x0A000939, 0x20010000, 0x00000000, 0x00000000, 0x00000000,
    0x00000080, 0x00000000, 0x00050100, 0x001000F6, 0x00000000,
    0x00000000, 0x00000000, 0x00000000, 0xFF000000,
};
```